plugin/src/main/resources/seedJobGroovyScript.groovy

```diff
@@ -119,28 +118,22 @@ def updateJobs(FilePath filePath, String namePrefix = '') {
 def workspace = build.workspace
 def xmlBase = new FilePath(workspace, 'XML_BASE_DIR_PLACEHOLDER')
 
-def localFile = new File("TEMPORARY_XML_DIR_PREFIX_PLACEHOLDER-${build.getNumber()}")
-println "Create local XML dir '${localFile.absolutePath}'"
-if (localFile.exists()) {
-    throw new RuntimeException("Local XML dir '${localFile.absolutePath}' already exists")
-}
-if (!localFile.mkdir()) {
-    throw new RuntimeException("Could not create local XML dir '${localFile.absolutePath}'")
-}
-def localXmlBase = new FilePath(localFile)
+def localDir = Files.createTempDirectory("TEMPORARY_XML_DIR_PREFIX_PLACEHOLDER")
+println "Created local XML dir '${localDir}'"
+def localXmlBase = new FilePath(new File(localDir.toString()))
```

Comment:
You can use `new FilePath(localDir.toFile())` instead.

Reply:
Done